pandas/core/internals/managers.py

```diff
@@ -247,6 +247,16 @@ def _has_no_reference_block(self, blkno: int) -> bool:
         """
         return not self.blocks[blkno].refs.has_reference()
 
+    def add_references(self, mgr: BaseBlockManager) -> None:
+        """
+        Adds the references from one manager to another.
```

Comment:
This assumes an identical block structure between both managers?

Reply:
Yep, I’ll add a comment explaining this

Reply:
Added